integration.md

```diff
@@ -0,0 +1,104 @@
+**Writing and Running Integration Tests**
+
+This article explores steps to write and run integration tests for Kubebuilder.
+
+Based on an example controller that watches *Parent* objects that in turn create *Child* objects, Kubebuilder provides framework for writing integration tests under `pkg/controller/parent/parent_controller_test.go`. Below is a detailed example of the file:
+```
```

Comment:
We can improve the clarity if we use some *real* resources for parent and child. Consider using `ReplicaSet` as Parent and `Pod` as child.

Reply:
I understand all of that you said above. Using some concrete resource examples are bit easier to understand. Also, I realized that when it comes to testing, testing core resources should not be any different from testing CRD resources :)

Don't have any strong opinion, so lets go ahead with the merge.

Reply:
> testing core resources should not be any different from testing CRD resources
Agreed, but we should also think what is the focus of kubebuilder, which (in my opinion) is about reconciliating custom resources, not core resources. Just my 2 cents. :D